### DISCUSSION TITLE
Configuration File

### DISCUSSION THREAD

**Discussion by kdawson133** [wezterm/wezterm#3105] [Q&A]
is there a way for setting to be applied by target triple?

**Comment by aorith** [comment]
I'm not sure if this is what you mean but the configuration table doesn't need to be returned immediately.

You can create a table and apply configuration conditionally.

For example: https://github.com/aorith/dotfiles/blob/096de1590f0c3ad90153447e1c9a97ae70fbd162/topics/wezterm/src/wezterm/wezterm.lua#L61

**Comment by kdawson133** [comment]
Thanks, it appears to do the trick!
